script/Arcueid/Day 1/01_00_ARC01_2.txt

```diff
@@ -346,7 +344,7 @@ If you want to live an unprincipled life, find some like-minded people on the st
 [sha:f1de3fd9e9fa91a6459ee54680a5116458d025f1]{
 -- Page 93, Offset 1405.
 -- いや、中学からこっち総額一万円近い金額を貸したきり戻ってこないので、外敵と言えば間違いなく外敵だけど。
-No, I've lent him almost 10,000 yen since middle school and he hasn't given any of it back, so I can call him a foreign enemy at this point.
+No, I've lent him almost 10,000 yen since middle school and he hasn't given any of it back, so that's no longer a rival, just a normal enemy.
```

Comment:
any of it back. That's not a rival anymore, that's just a normal enemy.

Reply:
Oh this is good, it's better than what I came up with.